cdc/scheduler/internal/v3/replication/replication_manager_test.go

```diff
@@ -652,11 +652,22 @@ func TestReplicationManagerAdvanceCheckpoint(t *testing.T) {
 	require.Equal(t, model.Ts(10), checkpoint)
 	require.Equal(t, model.Ts(20), resolved)
 
-	// some table not exist yet.
+	// some table not exist yet with redo is enabled.
 	currentTables.UpdateTables([]model.TableID{1, 2, 3})
-	checkpoint, resolved = r.AdvanceCheckpoint(currentTables, time.Now(), schedulepb.NewBarrierWithMinTs(30), redoMetaManager)
+	barrier := schedulepb.NewBarrierWithMinTs(30)
```

Comment:
Does the testing here reproduce the customer's problem? If so, maybe we should test redo scenior in a separate function, such as `TestReplicationManagerAdvanceCheckpointWithRedo`.